pom.xml

```diff
@@ -88,7 +88,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <ramlfiles_path>${basedir}/ramls/</ramlfiles_path>
-    <raml-module-builder-version>29.4.0</raml-module-builder-version>
+    <raml-module-builder-version>29.5.0-SNAPSHOT</raml-module-builder-version>
```

Comment:
@MikhailFokanov  @julianladisch  @adamdickmeiss Are we intending to run mod-inventory-storage master on a SNAPSHOT version of RAML Module Builder temporarily or permanently?

Reply:
This is temporary for testing. I will remove it after the build will be green. I should have posted a comment.

Reply:
> I should have posted a comment.

That's ok.

> This is temporary for testing. I will remove it after the build will be green. 

After which build goes green, master or this pull request?

@skoczko mentioned a week or so ago that master would need to run a SNAPSHOT build temporarily in order for the performance tests to run against the new record count changes.

Is that no longer the plan?

Reply:
I've merged this to master to run the performance test as discussed on https://issues.folio.org/browse/MODINVSTOR-468 .